src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -1,43 +1,69 @@
 import { notify } from 'react-notify-toast';
-import { addDictionary, fetchOrganizations, isSuccess, isFetching, isErrored } from './dictionaryActions';
+import {
+  addDictionary,
+  fetchOrganizations,
+  isSuccess,
+  isFetching,
+  isErrored,
+} from './dictionaryActions';
 import api from './../../api';
 
+/* eslint-disable */
 export const createDictionary = data => dispatch =>
   api.dictionaries
     .createDictionary(data)
     .then(payload => dispatch(
       addDictionary(payload),
-      notify.show('Successfully added dictionary to your organization', 'success', 6000),
+      notify.show(
+        'Successfully added dictionary to your organization',
+        'success', 6000,
+      ),
     ))
-    .catch(error => /* eslint-disable */
+    .catch(error =>
       notify.show(`${error.response.data.__all__[0]}`, 'error', 6000));
 
 export const createDictionaryUser = data => dispatch =>
   api.dictionaries
     .createDictionaryUser(data)
     .then(payload => dispatch(
       addDictionary(payload),
-      notify.show('Successfully added your dictionary', 'success', 6000),
+      notify.show(
```

Comment:
why add this space

Reply:
it is not space @hadijahkyampeire it was a linting error fixture.